app/models/organization/OrganizationService.scala

```diff
@@ -115,7 +115,7 @@ class OrganizationService @Inject()(organizationDAO: OrganizationDAO,
     def sendRPCToDataStore(dataStore: DataStore) =
       rpc(s"${dataStore.url}/data/triggers/newOrganizationFolder")
         .addQueryString("token" -> dataStoreToken, "organizationName" -> organizationName)
-        .post
+        .post()
```

Comment:
?

Reply:
scala style guide recommends adding () to parameter-less function calls iff the function contain side effects. Intellij reminded me that this is the case here